packages/mesa/build.sh

```diff
@@ -3,11 +3,11 @@ TERMUX_PKG_DESCRIPTION="An open-source implementation of the OpenGL specificatio
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_LICENSE_FILE="docs/license.rst"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="24.2.8"
+TERMUX_PKG_VERSION="24.3.1"
```

Comment:
Can you bump this to 24.3.3 if we're here already?
See: https://repology.org/project/mesa/information

Reply:
Sure! I should have checked myself, thanks!